packages/synapse-react-client/src/synapse-queries/file/UploadToS3.test.ts

```diff
@@ -0,0 +1,79 @@
+import { Upload } from '@aws-sdk/lib-storage'
+import { uploadToS3 } from './UploadToS3'
+
+jest.mock('@aws-sdk/lib-storage', () => {
```

Comment:
I originally tried to mock the S3 HTTP API using Mock Service Worker. Unfortunately, I ran into issues where the AWS SDK required methods on `Blob` that JSDOM does not polyfill, and replacing the `Blob` polyfill caused errors in other tests because other JSDOM polyfills rely on using their `Blob` polyfill.